### PR TITLE
Release 2.52.0

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.52.0](https://github.com/serverless/serverless/compare/v2.51.2...v2.52.0) (2021-07-15)
+
+### Features
+
+- **AWS CloudFormation:** Allow to disable default export names ([#9748](https://github.com/serverless/serverless/pull/9748)) ([6f49488](https://github.com/serverless/serverless/commit/6f494888cc01853894ec33859edbd77a06dc9d76)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **CLI:** Automatically expand loaded env variables ([#9615](https://github.com/serverless/serverless/issues/9615)) ([1864969](https://github.com/serverless/serverless/commit/186496922a0c4d69f3101dde0a9f4a0d89995ad0)) ([David Asensio Cañas](https://github.com/d-asensio))
```
Comment (medikoo) [inline CHANGELOG.md:10]: Let's indicate that it applies to environment variables as loaded from `.env` files.

Current description is not clear

**Review by medikoo — COMMENTED** [review]
@pgrzesik looks good. I just suggest to expand a bit info on `.env` extension

File: CHANGELOG.md (hunk 1 above)
Comment (pgrzesik) [reply CHANGELOG.md]: :+1: good suggestion

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**